CE07SHSM/CE07SHSM_D00018_ingest.csv

```diff
@@ -0,0 +1,37 @@
+# mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CE07SHSM/D00018/cg_data/dcl37/syslog/*.syslog.log,CE07SHSM-MFD37-00-DCLENG000,telemetered,Pending,Incorrectly specified driver and dataset
+mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CE07SHSM/D00018/cg_data/dcl37/optaa*/*.optaa*.log,CE07SHSM-MFD37-01-OPTAAD000,telemetered,Available,
+mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_telemetered_driver,/omc_data/whoi/OMC/CE07SHSM/D00018/cg_data/dcl37/ctdbp2/*.ctdbp*.log,CE07SHSM-MFD37-03-CTDBPC000,telemetered,Available,
+mi.dataset.driver.dosta_abcdjm.ctdbp.dcl.dosta_abcdjm_ctdbp_dcl_telemetered_driver,/omc_data/whoi/OMC/CE07SHSM/D00018/cg_data/dcl37/ctdbp2/*.ctdbp*.log,CE07SHSM-MFD37-03-DOSTAD000,telemetered,Available,
+mi.dataset.driver.zplsc_c.dcl.zplsc_c_dcl_telemetered_driver,/omc_data/whoi/OMC/CE07SHSM/D00018/cg_data/dcl37/zplsc/*zplsc*.log,CE07SHSM-MFD37-07-ZPLSCC000,telemetered,Available,
+# ,/omc_data/whoi/OMC/CE07SHSM/D00018/,CE07SHSM-MFD37-06-CAMDSA000,telemetered,Pending,No parser exists for this data set
+# mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_telemetered_driver,/omc_data/whoi/OMC/CE07SHSM/D00018/cg_data/cpm2/syslog/cpm_status.txt,CE07SHSM-RIC21-00-CPMENG000,telemetered,Pending,Incorrectly specified driver and dataset
+# mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CE07SHSM/D00018/cg_data/dcl26/syslog/*.syslog.log,CE07SHSM-RID26-00-DCLENG000,telemetered,Pending,Incorrectly specified driver and dataset
+mi.dataset.driver.adcpt_acfgm.dcl.pd0.adcpt_acfgm_dcl_pd0_telemetered_driver,/omc_data/whoi/OMC/CE07SHSM/D00018/cg_data/dcl26/adcpt1/*.adcpt*.log,CE07SHSM-RID26-01-ADCPTA000,telemetered,Available,
+mi.dataset.driver.velpt_ab.dcl.velpt_ab_dcl_telemetered_driver,/omc_data/whoi/OMC/CE07SHSM/D00018/cg_data/dcl26/velpt2/*.velpt*.log,CE07SHSM-RID26-04-VELPTA000,telemetered,Available,
+mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_telemetered_driver,/omc_data/whoi/OMC/CE07SHSM/D00018/cg_data/dcl26/phsen1/*.phsen*.log,CE07SHSM-RID26-06-PHSEND000,telemetered,Available,
+mi.dataset.driver.suna.suna_dcl_driver,/omc_data/whoi/OMC/CE07SHSM/D00018/cg_data/dcl26/nutnr/*.nutnr.log,CE07SHSM-RID26-07-NUTNRB000,telemetered,Available,
+mi.dataset.driver.spkir_abj.dcl.spkir_abj_dcl_telemetered_driver,/omc_data/whoi/OMC/CE07SHSM/D00018/cg_data/dcl26/spkir/*.spkir.log,CE07SHSM-RID26-08-SPKIRB000,telemetered,Available,
+# mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CE07SHSM/D00018/cg_data/dcl27/syslog/*.syslog.log,CE07SHSM-RID27-00-DCLENG000,telemetered,Pending,Incorrectly specified driver and dataset
+mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CE07SHSM/D00018/cg_data/dcl27/optaa1_dec*/*.optaa*.log,CE07SHSM-RID27-01-OPTAAD000,telemetered,Available,
```

Comment:
Yet here it is aware of the "optaa_decimated"

Reply:
I'll correct this one to match the others.